packages/gatsby-source-wordpress/package.json

```diff
@@ -1,53 +1,102 @@
+  "version": "4.0.0",
   "bugs": {
     "url": "https://github.com/gatsbyjs/gatsby/issues"
   },
-  "bundledDependencies": [],
   "dependencies": {
-    "@babel/runtime": "^7.12.5",
+    "@babel/runtime": "^7.6.3",
+    "@rematch/core": "^1.3.0",
+    "@rematch/immer": "^1.2.0",
+    "@types/cache-manager": "^2.10.3",
+    "@types/ink": "^2.0.3",
+    "async-retry": "^1.3.1",
+    "atob": "^2.1.2",
     "axios": "^0.21.1",
+    "axios-rate-limit": "^1.3.0",
     "better-queue": "^3.8.10",
-    "bluebird": "^3.7.2",
-    "deep-map": "^1.5.0",
-    "deep-map-keys": "^1.2.0",
-    "gatsby-source-filesystem": "^2.12.0-next.0",
-    "json-stringify-safe": "^5.0.1",
-    "lodash": "^4.17.20",
-    "minimatch": "^3.0.4",
-    "qs": "^6.9.4"
+    "btoa": "^1.2.1",
+    "cache-manager": "^3.4.0",
+    "cache-manager-fs-hash": "^0.0.9",
+    "chalk": "^4.1.0",
+    "cheerio": "^1.0.0-rc.3",
+    "clipboardy": "^2.1.0",
+    "diff": "^5.0.0",
+    "dumper.js": "^1.3.1",
+    "execall": "^2.0.0",
+    "fast-json-stable-stringify": "^2.1.0",
+    "file-type": "^15.0.1",
+    "filesize": "^6.1.0",
+    "fs-extra": "^9.0.1",
+    "gatsby-core-utils": "1.11.0-next.0",
+    "gatsby-image": "2.12.0-next.0",
+    "gatsby-plugin-catch-links": "2.11.0-next.0",
+    "gatsby-source-filesystem": "2.12.0-next.0",
+    "glob": "^7.1.6",
+    "got": "^11.7.0",
+    "graphql-query-compress": "^1.2.2",
+    "lodash": "^4.17.19",
+    "node-fetch": "^2.6.0",
+    "p-queue": "^6.3.0",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "read-chunk": "^3.2.0",
+    "replaceall": "^0.1.6",
+    "semver": "^7.3.2",
+    "valid-url": "^1.0.9"
   },
-  "deprecated": false,
   "devDependencies": {
-    "@babel/cli": "^7.12.1",
-    "@babel/core": "^7.12.3",
-    "babel-preset-gatsby-package": "^0.13.0-next.0",
-    "cross-env": "^7.0.3"
+    "@babel/cli": "^7.6.4",
+    "@babel/core": "^7.6.4",
+    "@babel/plugin-proposal-class-properties": "^7.10.1",
+    "@babel/plugin-proposal-private-methods": "^7.10.1",
+    "@types/semver": "^7.3.4",
+    "babel-jest": "^25.1.0",
+    "babel-plugin-import-globals": "^2.0.0",
+    "babel-plugin-module-resolver": "4.0.0",
+    "babel-preset-gatsby": "0.13.0-next.0",
+    "babel-preset-gatsby-package": "0.13.0-next.0",
+    "cross-env": "^5.2.1",
+    "identity-obj-proxy": "^3.0.0",
+    "jest": "^25.1.0",
+    "react-test-renderer": "^16.12.0",
+    "rimraf": "^3.0.2",
+    "tree-kill": "^1.2.2",
+    "wait-on": "^4.0.0"
   },
-  "homepage": "https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-source-wordpress#readme",
+  "homepage": "https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-source-wordpress",
   "keywords": [
     "gatsby",
     "gatsby-plugin",
-    "gatsby-source-plugin"
+    "gatsby-plugin-wordpress",
+    "gatsby-source-wordpress",
+    "wordpress"
   ],
   "license": "MIT",
   "peerDependencies": {
-    "gatsby": "^2.0.15"
+    "gatsby": "^2.24.74",
```

Comment:
Do you actually need 2.24.74? or a lower version also okay?

Reply:
We need this version because of the plugin options schema API but also because earlier versions of Gatsby had develop state machine bugs that completely broke this plugin